app/helpers/st_prov_details_helper.rb

```diff
@@ -0,0 +1,69 @@
+module StProvDetailsHelper
+  private
+
+  def row_data_with_title(label, title, value)
+    {:cells => {:label => label, :value => value, :title => title}}
+  end
+
+  def tab_label(tab)
+    {:name => tab.id, :text => _(tab.label)}
+  end
+
+  def st_prov_show_tab_configuration(work_flow)
+    tab_labels = []
+    work_flow.dialog.dialog_tabs.map do |tab|
+      tab_label(tab)
+    end
+    tab_labels
+  end
+
+  def st_prov_show_tab_content(key_name, tab_index, &block)
+    class_name = tab_index == 0 ? 'tab_content active' : 'tab_content'
+    tag.div(:id => key_name, :class => class_name, &block)
+  end
+
+  def st_prov_show_tab_info(tab, work_flow)
+    summary = []
+    tab.dialog_groups.each do |group|
+      rows = []
+      unless group.dialog_fields.empty?
+        group.dialog_fields.each do |field|
+          rows.push(request_dialog_details(work_flow, field))
+        end
+      end
+      summary.push(miq_structured_list(
+                     :title => _(group.label),
```

Comment:
@DavidResende0 , could you please check if the translations work here?

Reply:
By default, just applying _() to a variable will not actually mark the string for translation since the check for which strings need translations occurs before any code in the frontend is run.

The solution is to ensure that the actual string in the backend is surrounded by `N_()` which saves the translated string to the db but doesn't actually translate it until _() is called in the frontend later.

From what I can tell though, this string seems like it is generated by the user when they create their dialog. If that's the case then it shouldn't be translated anyways. Not 100% sure though if it is user generated.